src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1617,6 +1617,24 @@ public boolean areThereResponsesForQuestion(String feedbackQuestionId) {
         return feedbackQuestionsLogic.getCopiableFeedbackQuestionsForInstructor(googleId);
     }
 
+    /**
+     * Generates template feedback questions for an instructor in the session.<br>
+     * Returns an empty list if there are no questions for the template
+     * for the session.
+     * Preconditions: <br>
+     * * All parameters are non-null.
+     */
+    public List<FeedbackQuestionAttributes> populateFeedbackSessionTemplateQuestions(String templateType, String courseId,
+                                                                                     String feedbackSessionName,
```

Comment:
For long method name, we are getting rid of indenting like this. 8 space indentation should be enough.